WireUI/Sources/WireConversationUI/Resouces/UImage+ImageResource.swift

```diff
@@ -15,3 +15,9 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
+
+import UIKit
+
+public extension UIImage {
```

Comment:
question:
Do extensions like this provide a lot of value. In my opinion we're spoiling the namespace of `UIImage`. Also if the image is only used once, it's actually less code to write
```
UIImageView(image: UIImage(resource: .dropdown))
```
than having this extension.

suggestion: if we keep it, please make it internal, until we really need it outside the target.

```suggestion
extension UIImage {
```

Reply:
I think this is more about approach, I made it similar to `AccountImageView`.  For me it's just syntax sugar, it's convenient and provides growth possibilities for whole `WireConversationUI` module where eventually would be more images. 
I'm ok to follow any style team agreed and got used to. Did you had this conversation before or it's better to ask the team?